src/main/webapp/WEB-INF/hw/task1/chemicalElements.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html xmlns:margin="http://www.w3.org/1999/xhtml">
+    <head>
+        <style>
+            div {
+                width: 30px;
+                margin-left: auto;
+                margin-right: auto;
+            }
+            a {
+                color: blue;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>
+            Chemical elements
+        </h1>
+        <hr>
+        <table>
+            <tr>
+                <td>Number</td>
+                <td>Name</td>
+                <td>Symbol</td>
+                <td>Year</td>
+                <td>Article</td>
+            </tr>
+            <tr>
+                <td>1</td>
+                <td>
+                    <a href="https://ru.wikipedia.org/wiki/%D0%92%D0%BE%D0%B4%D0%BE%D1%80%D0%BE%D0%B4">Hydrogen</a>
+                </td>
+                <td>
+                    <strong>H</strong>
+                </td>
+                <td>1766</td>
+                <td>
+                    <div width="30px">
+                        <a href="https://ru.wikipedia.org/wiki/%D0%92%D0%BE%D0%B4%D0%BE%D1%80%D0%BE%D0%B4">
+                            <img src="/resources/article_image.png" alt="article_image" style="width:30px; height:30px" class="center">
+                        </a>
+                    </div>
+                </td>
+            </tr>
+            <tr>
+                <td>2</td>
+                <td>
+                    <a href="https://ru.wikipedia.org/wiki/%D0%93%D0%B5%D0%BB%D0%B8%D0%B9">Helium</a>
+                </td>
+                <td>
+                    <strong>He</strong>
+                </td>
+                <td>1895</td>
+                <td>
+                    <div>
+                        <a href="https://ru.wikipedia.org/wiki/%D0%93%D0%B5%D0%BB%D0%B8%D0%B9">
+                            <img src="/resources/article_image.png" alt="article_image" style="width:30px; height:30px">
```

Comment:
If you move some common stiles to head, let's do the same for `img`

Reply:
@DHrytsuk Updated: [7cc84ffc](7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)